docs/user-guide/release-notes/2.10.x.rst

```diff
@@ -23,3 +23,5 @@ New Features
   the importer configuration. More details can be found in `RPM Features
   <http://docs.pulpproject.org/user-guide/features.html#package-signature-verification>`_ or in
   `Recipes <http://docs.pulpproject.org/user-guide/recipes.html#sync-a-repo-with-signature-verification>`_.
+
+* Added support for uploading DRPMs via `pulp-admin rpm repo uploads drpm` and via API. 
```

Comment:
seems like this RFE will go into 2.11, so please put the release notes into according file.